projects/laji/src/app/+theme/protax/protax-form/protax-form.component.ts

```diff
@@ -35,6 +38,11 @@ export class ProtaxFormComponent {
   }
 
   submitForm() {
+    if (this.probabilityThreshold == null || this.probabilityThreshold < 0 || this.probabilityThreshold > 1) {
```

Comment:
`this.probabilityThreshold == null` could be replaced with just `!this.probabilityThreshold` or if you need to check the null case specifically you need to use `this.probabilityThreshold === null`

Reply:
`!this.probabilityThreshold` would not work with 0 so I used `this.probabilityThreshold == null` to check that it isn't undefined or null

Reply:
true 👍